master/master.c

```diff
@@ -1968,7 +1968,7 @@ static void limit_fds(rlim_t x)
     struct rlimit rl;
 
 #ifdef HAVE_GETRLIMIT
-    if (!getrlimit(RLIMIT_NUMFDS, &rl)) {
+    if (getrlimit(RLIMIT_NUMFDS, &rl) >= 0) {
```

Comment:
`getrlimit` only returns `0` (on success) or `-1` (on error), so this is only a code style change -- and a misleading one at that, since it incorrectly implies that it might return a non-zero positive number sometimes.